solutions/toc.yml

```diff
@@ -458,15 +458,24 @@ toc:
           - file: observability/logs/logs-index-template-reference.md
             children:
               - file: observability/logs/logs-index-template-defaults.md
-          - file: observability/streams/streams.md
-            children:
-              - file: observability/streams/management/extract.md
-                children:
-                  - file: observability/streams/management/extract/date.md
-                  - file: observability/streams/management/extract/dissect.md
-                  - file: observability/streams/management/extract/grok.md
-              - file: observability/streams/management/retention.md
-              - file: observability/streams/management/advanced.md
+      - file: observability/streams/streams.md
```

Comment:
should we move this out of observability? It also applies to security (but it's linked in the a different place there)

Reply:
Is most of the functionality here still logs/observability? It seems like for the most part we are still discussing streams from an observability context. Maybe it makes sense to maybe link from the security docs to these docs?

Reply:
> sounds ok for me too, we don't have to rush moving this, but cross linking from Security and maybe even Stack Management would make sense

[This PR](https://github.com/elastic/docs-content/pull/3399) adds a few bits of information about streams from a data stream management perspective (from the [Manage data](https://www.elastic.co/docs/manage-data/) section of the docs). 
@LucaWintergerst, I'll add you as a reviewer, if you could please have a glance over it. Many thanks in advance!